feat(pie-docs): DSW-1259 added components L-P (#840)
justeattakeaway/pie: apps/pie-docs/src/components/numeric-stepper/numeric-stepper-form/numeric-stepper-form.md

---
eleventyNavigation:
    key: 'Numeric Stepper - Form'
    parent: numeric-stepper
---

## Overview

> The purpose of numeric steppers as form fields is to offer users flexibility in inputting numeric values, maintaining a consistent form field appearance in form interfaces. This allows users to easily adjust and enter numerical data with ease.

Reviewer: ```suggestion
The purpose of numeric steppers as form fields is to offer users flexibility in inputting numeric values, while maintaining a consistent form field appearance in form interfaces. This allows users to easily enter and adjust numerical data.
```

Author: this small change got me looking at this 😄 https://www.grammarly.com/blog/while-vs-whilst/#:~:text=Whilst%20and%20while%20are%20two,That's%20the%20main%20difference.

> Numeric steppers [form] resemble the structure of classic form fields, but allow for precise value entry through the integrated text input while also offering the convenience of incremental adjustments using the provided buttons.

Reviewer: This sentence is a bit wordy, in my opinion, we should be aiming for simpler words and phrases that are easy for anyone to understand on a Friday afternoon 😄

Author: I agree, what about something like:

`Numeric steppers [form] allow a user to enter precise values, either by inputting text or by using the increment or decrement buttons.`

Reviewer: Yeah that's great!
